main/index.html

```diff
@@ -138,12 +138,12 @@ <h1 class="title1">MetWork Framework</h2>
                 </div>
                 <!-- layer 2 -->
                 <div class="layer-1-2 wow slideInUp" data-wow-duration="2s" data-wow-delay=".1s">
-                  <h2 class="title2">An opensource framework for building Linux production systems in Meteorology, IoT or other domains.</h1>
+                  <h2 class="title2">Save time in building, running and monitoring your applications</h1>
```

Comment:
```suggestion
                  <h2 class="title2">Save time in building, running and monitoring your Linux production systems.</h1>
```

Reply:
Ok I change my mind to your proposal. I just add some details about too vague "applciations" as MetWork is Linux only and is "server only"